tests/unit/test_templatetags.py

```diff
@@ -43,6 +44,22 @@ def test_get_translations(self, languages):
         language_codes = [l.code for l in translations.keys()]
         assert language_codes[0] == 'en'
         assert language_codes[1] == 'es'
-        assert language_codes[2] == 'fr' 
+        assert language_codes[2] == 'fr'
         assert language_codes[3] == 'de'
         assert language_codes[4] == 'nl'
+
+
+@pytest.mark.django_db
+class TestTranslationWagtailAdminTags(object):
```

Comment:
When using a single test case you don't need a test class, a method will suffice.
Can you add a test for when the templatetag returns `False`? so for example when a non-canonical page is sent to the tag. or `SYNC_TREE` is `False` Thanks! 😄